packages/core/src/logger.ts

```diff
@@ -76,23 +85,23 @@ export class Logger {
     return { write, metadata, level, options };
   }
 
-  error(msg: string, data?: Record<string, any> | Error): void {
+  error(msg: string, data?: Record<string, any> | unknown[] | unknown | Error): void {
```

Comment:
Any reason we changed the type signature here? What use cases do you need to pass an array?